third_party/zap/templates/print-cluster.zapt

```diff
@@ -0,0 +1,30 @@
+{{chip_header}}
+
+
+// Enclosing macro to prevent multiple inclusion
+#ifndef SILABS_PRINT_CLUSTER
+#define SILABS_PRINT_CLUSTER
+
+
+// This is the mapping of IDs to cluster names assuming a format according
+// to the "EmberAfClusterName" defined in the ZCL header.
+// The names of clusters that are not present, are removed.
+
+{{#zcl_clusters}}
+#if defined(ZCL_USING_{{asDelimitedMacro this.define}}_SERVER) || defined(ZCL_USING_{{asDelimitedMacro this.define}}_CLIENT)
+    #define SILABS_PRINTCLUSTER_{{asDelimitedMacro this.define}} {ZCL_{{asDelimitedMacro this.define}}_ID, {{this.code}}, "{{this.label}}" },
```

Comment:
Should we replace `SILABS` by `CHIP` in this file ? It's not a big deal since my understanding of this code is that the macro names are just used to generate `CLUSTER_IDS_TO_NAMES` which is used itself by `src/app/util/util.cpp` but I'm wondering since I believe the original idea was to prefix the macro to avoid name collision.

Reply:
done